product/toolbars/containers_center_tb.yaml

```diff
@@ -52,6 +52,12 @@
     :title: Monitoring
     :text: Monitoring
     :items:
+    - :button: container_timeline
+      :image: timeline
+      :text: "Timelines"
+      :title: "Show Timelines for this Container"
+      :url: '/explorer'

```

Comment:
@dkorn this warning message does seem to be related to timelines at all. did you restart server after removing :url: '/explorer', i am curious how is it going thru x_button method if you are sending up url as explorer. other than this comment other changes look good.